chart/values.yaml

```diff
@@ -2399,17 +2427,21 @@ httpCache:
   ##
   podSecurityContext:
     enabled: true
-    fsGroup: 0
+    fsGroup: 102
```

Comment:
Hi @FraPazGal I've used the default user for the base images (nginx 101:101, varnish 101:102, node 1000:1000). this will be as is meanwhile we adapt our containers to be rootless.